earth_enterprise/rpms/build.gradle

```diff
@@ -9,123 +9,265 @@ buildscript {
+    from(stagedInstallDir_manual) {
+        into packageInstallRootDir
     }
 }
 
-task serverRpm (type: Rpm) {
-    requires('opengee-common', packageVersion, GREATER | EQUAL)
+task openGeeServerRpm (type: Rpm) {
+    requires('opengee-common', openGeeVersion, GREATER | EQUAL)
     packageName = 'opengee-server'
     
-    from(staged_install_server_opt) {
-        into package_install_path + '/opt'
+    from(stagedInstallDir_server_opt) {
+        into new File(packageInstallRootDir, 'opt')
     }
-    from(staged_install_server_etc) {
-        into package_install_path + '/etc'
+    from(stagedInstallDir_server_etc) {
+        into new File(packageInstallRootDir, 'etc')
     }
 
-    from(staged_install_server_usermagic) {
-        into package_install_path
+    from(stagedInstallDir_server_usermagic) {
+        into packageInstallRootDir
     }
-    from(staged_install_server_apachesupport) {
-        into package_install_path
+    from(stagedInstallDir_server_apachesupport) {
+        into packageInstallRootDir
     }
 }
 
-task serverDeb (type: Deb) {
-    requires('opengee-common', packageVersion, GREATER | EQUAL)
+task openGeeServerDeb (type: Deb) {
+    requires('opengee-common', openGeeVersion, GREATER | EQUAL)
     packageName = 'opengee-server'
-    from(staged_install_server_etc) {
-        into package_install_path + '/etc'
+    from(stagedInstallDir_server_etc) {
+        into new File(packageInstallRootDir, 'etc')
     }
 
-    from(staged_install_server_opt) {
-        into package_install_path + '/opt'
+    from(stagedInstallDir_server_opt) {
+        into new File(packageInstallRootDir, 'opt')
     }
 
-    from(staged_install_server_apachesupport) {
-        into package_install_path
+    from(stagedInstallDir_server_apachesupport) {
+        into packageInstallRootDir
     }
 
-    from(staged_install_server_usermagic) {
-        into package_install_path
+    from(stagedInstallDir_server_usermagic) {
+        into packageInstallRootDir
     }
 }
 
-task fusionRpm (type: Rpm) {
-    requires('opengee-common', packageVersion, GREATER | EQUAL)
+task openGeeFusionRpm (type: Rpm) {
+    requires('opengee-common', openGeeVersion, GREATER | EQUAL)
     packageName = 'opengee-fusion'
-    from(staged_install_fusion) {
-        into package_install_path
+    from(stagedInstallDir_fusion) {
+        into packageInstallRootDir
     }
 }
 
-task fusionDeb (type: Deb) {
-    requires('opengee-common', packageVersion, GREATER | EQUAL)
+task openGeeFusionDeb (type: Deb) {
+    requires('opengee-common', openGeeVersion, GREATER | EQUAL)
     packageName = 'opengee-fusion'
-    from(staged_install_fusion) {
-        into package_install_path
+    from(stagedInstallDir_fusion) {
+        into packageInstallRootDir
     }
 }
 
-task packageRpms(dependsOn: ['serverRpm','fusionRpm','commonRpm'])
-task packageDebs(dependsOn: ['serverDeb','fusionDeb','commonDeb'])
-task osPackage(dependsOn: ['packageRpms','packageDebs'])
+task opengeeRpms(
```

Comment:
Not sure if this matters but there's a capitalization inconsistency here - the other targets start with openGee, not opengee.  Same with opengeeDebs below.